sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -1,18 +1,12 @@
 # Release History
 
-## 1.4.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.4.0-beta.2 (2024-09-11)
 
 ### Bugs Fixed
 
 * RPC attributes are now correctly exported to Application Insights as custom properties.
```

Comment:
the only change since last release is the one bug fix, but that too is partially fixed only as per https://github.com/Azure/azure-sdk-for-net/issues/45288#issuecomment-2276355033.
Wondering why this release is needed right now, without fixing the above as well?

Reply:
This is ongoing work but was put on hold while addressing some high priority items.
I felt releasing the partial fix was better than no fix.

Reply:
IMHO, its best to fully fix it, rather than shipping half-fixed solution.